package.json

```diff
@@ -8,6 +8,7 @@
   },
   "devDependencies": {
     "@gnosis.pm/mock-contract": "4.0.0",
+    "@gnosis.pm/safe-contracts": "^1.3.0",
```

Comment:
Are you using this?

Reply:
I'm trying to. That's where calculateProxyAddress seems to be defined but I'm still getting: 
`ReferenceError: calculateProxyAddress is not defined`.